issue_template.md

```diff
@@ -0,0 +1,19 @@
+## Issue description
+
+If you found a bug or want to propose a feature, please describe it as accurately as possible. Use the following checklists. This way we can help you the best. 
+
+### Your checklist for a bug report
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the issue. If you're unsure about any of them, don't hesitate to ask. We're here to help!_
+
+- [ ] Have you described how to reproduce your bug?
+- [ ] Which operating system, browser and versions do you use?
+- [ ] Can you attach some logs or screenshots to this issue?
+- [ ] Have you already tried solve it yourself? Can you provide any findings?
+
+### Your checklist for a feature request
+
+- [ ] Which problem should this feature solve? 
+- [ ] What is your specific requirement?
+- [ ] Can you implement this feature yourself? We will support you as much as possible.
```

Comment:
Maybe again concrete headings?

#### Problem to be solved
_Please tell, **what** you want to achieve, not how this should be done._

#### Proposed solution
_Here you can describe your proposed solution._